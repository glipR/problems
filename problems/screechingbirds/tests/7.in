50 281
21 67 fdyidsphrirxlemgodkzqlqb
15 35 hjfpwsmjmiwyo
33 76 siwnysbql
82 27 yrbjykxsgrig
49 22 pblbyjdzhulwiclzqrzxffty
70 24 mwaetl
64 68 qfxoqrrkbh
20 2 cmeyxrzihcekfasgap
54 59 kioedrbuow
54 60 kufctytujw
60 52 whuvhgqvojapdqnv
75 85 samhiklecfws
52 78 brtixhq
66 88 kvjbmwj
56 6 dmkmcotkghznqizmolqthbjuq
31 8 nsvwdwaqvptzz
2 57 dgdxbbqqkcsyywwzxohhux
62 50 vlocpylxericpvbf
100 63 faczgokauvs
21 59 uutlvwrmbrvmdesp
6 97 eyerkxjlgfopkp
88 79 fntnekncnackdhwv
52 37 mrqaibdgevwoa
54 10 lxgmqhceebnyagdafvouusoig
7 45 ifmmwilnf
37 90 owwxmicior
53 88 msnntwhgpoigysqoqblg
87 66 kqdpdp
11 74 ygcjhurpgwdqgkwvjkh
73 42 jeidbyeoo
38 83 chqumxrspehzdwzbduojyvzy
48 41 hdkzmqsqvnjshr
71 65 ciatlwgvhh
92 58 szwou
71 37 oakxrlnemxwzvkkbxrkqkyb
58 42 ffetjktvrsyulqx
40 31 cvzlekbwmv
57 69 rjcxdpiwyw
23 41 haerroczrzdmwjgqmim
60 67 jkjyrycz
45 40 gupshdj
89 57 niyaayottjcicchh
33 51 zjujqufnepvuwqfhddze
0 84 smjqubesexweree
70 20 gxiucxltxqfzddhnlomkjm
8 49 uypoi
68 76 qrilnwv
75 82 dolsd
51 6 uzlldey
63 24 mxpczrzkfv