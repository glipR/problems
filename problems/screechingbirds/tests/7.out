smjqubesexweree
dgdxbbqqkcsyywwzxohhux
eyerkxjlgfopkp
ifmmwilnf
uypoi
ygcjhurpgwdqgkwvjkh
hjfpwsmjmiwyo
cmeyxrzihcekfasgap
fdyidsphrirxlemgodkzqlqb
uutlvwrmbrvmdesp
cmeyxrzihcekfasgap
haerroczrzdmwjgqmim
cmeyxrzihcekfasgap
cmeyxrzihcekfasgap
cmeyxrzihcekfasgap
cmeyxrzihcekfasgap
nsvwdwaqvptzz
cmeyxrzihcekfasgap
siwnysbql
zjujqufnepvuwqfhddze
cmeyxrzihcekfasgap
cmeyxrzihcekfasgap
owwxmicior
cmeyxrzihcekfasgap
chqumxrspehzdwzbduojyvzy
nsvwdwaqvptzz
cmeyxrzihcekfasgap
cvzlekbwmv
cmeyxrzihcekfasgap
cmeyxrzihcekfasgap
gupshdj
cmeyxrzihcekfasgap
nsvwdwaqvptzz
cmeyxrzihcekfasgap
hdkzmqsqvnjshr
pblbyjdzhulwiclzqrzxffty
hjfpwsmjmiwyo
cmeyxrzihcekfasgap
uzlldey
cmeyxrzihcekfasgap
brtixhq
mrqaibdgevwoa
ifmmwilnf
msnntwhgpoigysqoqblg
cmeyxrzihcekfasgap
kioedrbuow
kufctytujw
lxgmqhceebnyagdafvouusoig
nsvwdwaqvptzz
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
rjcxdpiwyw
uypoi
uzlldey
cmeyxrzihcekfasgap
ffetjktvrsyulqx
dgdxbbqqkcsyywwzxohhux
cmeyxrzihcekfasgap
whuvhgqvojapdqnv
jkjyrycz
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
vlocpylxericpvbf
nsvwdwaqvptzz
uzlldey
mxpczrzkfv
qfxoqrrkbh
cmeyxrzihcekfasgap
lxgmqhceebnyagdafvouusoig
haerroczrzdmwjgqmim
cmeyxrzihcekfasgap
kvjbmwj
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
qrilnwv
uzlldey
mwaetl
cmeyxrzihcekfasgap
gxiucxltxqfzddhnlomkjm
pblbyjdzhulwiclzqrzxffty
nsvwdwaqvptzz
ciatlwgvhh
oakxrlnemxwzvkkbxrkqkyb
cvzlekbwmv
cmeyxrzihcekfasgap
jeidbyeoo
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
lxgmqhceebnyagdafvouusoig
samhiklecfws
dolsd
uzlldey
cmeyxrzihcekfasgap
cmeyxrzihcekfasgap
nsvwdwaqvptzz
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
uutlvwrmbrvmdesp
uzlldey
yrbjykxsgrig
cmeyxrzihcekfasgap
cmeyxrzihcekfasgap
lxgmqhceebnyagdafvouusoig
zjujqufnepvuwqfhddze
smjqubesexweree
hjfpwsmjmiwyo
ygcjhurpgwdqgkwvjkh
gupshdj
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
nsvwdwaqvptzz
kqdpdp
uzlldey
mxpczrzkfv
fdyidsphrirxlemgodkzqlqb
cmeyxrzihcekfasgap
fntnekncnackdhwv
mrqaibdgevwoa
hdkzmqsqvnjshr
niyaayottjcicchh
cmeyxrzihcekfasgap
gxiucxltxqfzddhnlomkjm
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
szwou
pblbyjdzhulwiclzqrzxffty
uzlldey
mwaetl
cmeyxrzihcekfasgap
lxgmqhceebnyagdafvouusoig
nsvwdwaqvptzz
cmeyxrzihcekfasgap
ifmmwilnf
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
uzlldey
cmeyxrzihcekfasgap
faczgokauvs
ffetjktvrsyulqx
cmeyxrzihcekfasgap
cvzlekbwmv
nsvwdwaqvptzz
eyerkxjlgfopkp
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
lxgmqhceebnyagdafvouusoig
haerroczrzdmwjgqmim
uzlldey
cmeyxrzihcekfasgap
uypoi
cmeyxrzihcekfasgap
oakxrlnemxwzvkkbxrkqkyb
siwnysbql
yrbjykxsgrig
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
gxiucxltxqfzddhnlomkjm
nsvwdwaqvptzz
uzlldey
mxpczrzkfv
cmeyxrzihcekfasgap
whuvhgqvojapdqnv
vlocpylxericpvbf
kioedrbuow
cmeyxrzihcekfasgap
kufctytujw
lxgmqhceebnyagdafvouusoig
pblbyjdzhulwiclzqrzxffty
jeidbyeoo
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
dgdxbbqqkcsyywwzxohhux
uzlldey
mwaetl
cmeyxrzihcekfasgap
nsvwdwaqvptzz
hjfpwsmjmiwyo
cmeyxrzihcekfasgap
chqumxrspehzdwzbduojyvzy
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
uzlldey
cmeyxrzihcekfasgap
lxgmqhceebnyagdafvouusoig
gupshdj
cmeyxrzihcekfasgap
mrqaibdgevwoa
rjcxdpiwyw
nsvwdwaqvptzz
owwxmicior
jkjyrycz
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
uzlldey
cmeyxrzihcekfasgap
brtixhq
hdkzmqsqvnjshr
gxiucxltxqfzddhnlomkjm
qfxoqrrkbh
cmeyxrzihcekfasgap
cvzlekbwmv
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
lxgmqhceebnyagdafvouusoig
nsvwdwaqvptzz
zjujqufnepvuwqfhddze
uzlldey
mxpczrzkfv
yrbjykxsgrig
cmeyxrzihcekfasgap
ciatlwgvhh
pblbyjdzhulwiclzqrzxffty
cmeyxrzihcekfasgap
uutlvwrmbrvmdesp
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
msnntwhgpoigysqoqblg
uzlldey
mwaetl
cmeyxrzihcekfasgap
ifmmwilnf
ffetjktvrsyulqx
nsvwdwaqvptzz
cmeyxrzihcekfasgap
lxgmqhceebnyagdafvouusoig
qrilnwv
oakxrlnemxwzvkkbxrkqkyb
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
haerroczrzdmwjgqmim
niyaayottjcicchh
uzlldey
cmeyxrzihcekfasgap
cmeyxrzihcekfasgap
szwou
gxiucxltxqfzddhnlomkjm
nsvwdwaqvptzz
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
kqdpdp
uzlldey
cmeyxrzihcekfasgap
kvjbmwj
lxgmqhceebnyagdafvouusoig
fdyidsphrirxlemgodkzqlqb
hjfpwsmjmiwyo
uypoi
cmeyxrzihcekfasgap
jeidbyeoo
dolsd
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
pblbyjdzhulwiclzqrzxffty
nsvwdwaqvptzz
ygcjhurpgwdqgkwvjkh
uzlldey
mxpczrzkfv
cmeyxrzihcekfasgap
samhiklecfws
cmeyxrzihcekfasgap
vlocpylxericpvbf
yrbjykxsgrig
faczgokauvs
mrqaibdgevwoa
cmeyxrzihcekfasgap
whuvhgqvojapdqnv
dmkmcotkghznqizmolqthbjuq
lxgmqhceebnyagdafvouusoig
cvzlekbwmv
gupshdj
uzlldey
mwaetl
cmeyxrzihcekfasgap
nsvwdwaqvptzz
fntnekncnackdhwv
cmeyxrzihcekfasgap
smjqubesexweree
cmeyxrzihcekfasgap
dmkmcotkghznqizmolqthbjuq
gxiucxltxqfzddhnlomkjm
hdkzmqsqvnjshr
